syncd/RequestShutdownCommandLineOptionsParser.cpp

```diff
@@ -23,7 +23,9 @@ std::shared_ptr<RequestShutdownCommandLineOptions> RequestShutdownCommandLineOpt
         { "cold", no_argument, 0, 'c' },
         { "warm", no_argument, 0, 'w' },
         { "fast", no_argument, 0, 'f' },
+        { "express", no_argument, 0, 'e' },
         { "pre",  no_argument, 0, 'p' }, // Requesting pre shutdown
+        { "pxe",  no_argument, 0, 'z' }, // Requesting pre-express shutdown
```

Comment:
maybe preexpress would be better ?

Reply:
I am fine with either way. But this is called in the express-reboot script. Let me know if you want the change. I will need to change the caller side too.